charmcraft/commands/utils.py

```diff
@@ -33,6 +39,20 @@ def make_executable(fh):
     os.fchmod(fileno, mode)
 
 
+def load_yaml(fpath):
+    """Return the content of a YAML file."""
+    if not fpath.exists():
```

Comment:
```suggestion
    if not fpath.is_file():
```

Reply:
Nice, improved (and added a test for it).